composition/src/errors/errors.ts

```diff
@@ -709,7 +709,7 @@ export function undefinedFieldInFieldSetErrorMessage(
 ): string {
   return (
     ` The following field set is invalid:\n  "${fieldSet}"\n` +
-    ` This is because of the selection set corresponding to the field coordinate "${parentTypeName}.${fieldName}".\n` +
+    ` This is because of the selection set corresponding to the Field coordinates "${parentTypeName}.${fieldName}".\n` +
```

Comment:
Why is field capitalized?

Reply:
GraphQL jargon like Object, Interface, Enum, etc. are capitalised in our messages now. If they're not, it's because I missed them.